plugins/module_utils/robot.py

```diff
@@ -66,7 +74,7 @@ def plugin_open_url_json(plugin, url, method='GET', timeout=10, data=None, heade
         raise PluginException('Failed request to Hetzner Robot server endpoint {0}: {1}'.format(url, e))
 
     if not content:
-        if allow_empty_result and status in (200, 204):
+        if allow_empty_result and status in (200, 201, 204):
```

Comment:
I think it would be better to allow to provide a list/set of allowed status codes. (If not specified, the list should be set to `(200, 204)`.) That way every user of this (and the other) function can specify which codes they accept and which they do not accept.

Reply:
Solved in 2c357753432436dad8d07cec0d737718ad388234.